### ISSUE TITLE
displaymessage for twitchbot

### ISSUE THREAD

**Issue by neyoneit** [JBustos22/oDFe#9]
Hey,

we all know that displaymessage had to be disabled for twitchbot, due to fact that maps have slurs, racism etc in them.
Yesterday there was discussion on discord about making it on custom chsinfo, which would work, but we would have to move angles, speedometer, obs to chsinfo2 and it would be simply bad for viewers.

What i would propose to make displaymessage alike function, but it would be triggered only with twitchbot.
Basically the same function, but it could not be triggered and visible only by player who triggered it (twitchbot)

Best regards,
neit :)

**Comment by Jelvan1** [comment]
This logic also resides in the defrag mod (cfr. #17). To satisfy @neyoneit's needs, a temporary defrag mod patch was created.

I propose to close this issue.

**Comment by neyoneit** [comment]
Jel is da best Jel is greatest <3
